src/main/java/com/leverx/learningmanagementsystem/coursesettings/model/CourseSettings.java

```diff
@@ -24,7 +24,9 @@
 public class CourseSettings {
 
     @Id
-    @GeneratedValue(strategy = UUID)
+    @GeneratedValue
+    @JdbcTypeCode(SqlTypes.VARCHAR)
+    @Column(length = 36, nullable = false, updatable = false)
```

Comment:
why these constraints are required here? we are using liquibase for database structure creation